src/pages/FavDetails/FavDetails.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {render, screen} from '@testing-library/react'
+import FavDetails from '../FavDetails'
+import { BrowserRouter } from "react-router-dom/cjs/react-router-dom.min";
+
+
+test('loads and displays placeholder text', async () => {
+    render(
+        <BrowserRouter>
+            <FavDetails />
+        </BrowserRouter>);
+ 
+    expect(screen.getByTitle('placeholder')).toHaveTextContent('fav details page (private)')
```

Comment:
Oh I see your usage if you remove placeholder you can do this, 
```suggestion
    expect(screen.getByText('fav details page (private)')).toBeInTheDocument();
```